src/providers/bedrock/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const BEDROCK_STABILITY_V2_MODELS = [
```

Comment:
Considering that v1 models are now deprecated, should we maintain a list of those models instead of maintaining v2 models? Because v2 models might keep increasing as they are the active version.

Reply:
Or a better solution as per me would be to maintain naming patterns. For example, anything that contains `sd3` in it is the new model